portfolio/src/main/webapp/newScript.js

```diff
@@ -0,0 +1,11 @@
+/**Fethces the response from the server and adds it to the page */
+async function showResponse() {
+    const responseFromServer = await fetch('/hello');
+    const myObject = await responseFromServer.json();
```

Comment:
How about a more descriptive name for this object? Something like response?

Reply:
Gotcha!